jetty-core/jetty-util/src/main/java/org/eclipse/jetty/util/resource/Resource.java

```diff
@@ -72,135 +67,41 @@ public abstract class Resource implements ResourceFactory
         .with("jar:")
         .build();
 
-    /**
-     * <p>Create a ResourceCollection from an unknown set of URIs.</p>
-     *
-     * <p>
-     *     Use this if you are working with URIs from an unknown source,
-     *     such as a user configuration.  As some of the entries
-     *     might need mounting, but we cannot determine that yet.
-     * </p>
-     *
-     * @param uris collection of URIs to mount into a {@code ResourceCollection}
-     * @return the {@link Mount} with a root pointing to the {@link ResourceCollection}
-     */
-    public static Resource.Mount mountCollection(Collection<URI> uris)
```

Comment:
Don't forget to remove the ResourceCollection.Mount too.

Reply:
Done.